scripts/update_json_docs.sh

```diff
@@ -23,18 +29,25 @@ function pushDocs () {
     echo "Nothing to commit."
   fi
   cd ..
+
+  echo "Done deploying JSON documentation."
 }
 
 function cleanSubmodule () {
-    echo "Cleaning up..."
+    echo "Cleaning up!"
     git submodule deinit -f ghpages
     git reset HEAD .gitmodules
     git reset HEAD ghpages
     rm -rf ghpages
     rm -f .gitmodules
     rm -rf .git/modules/ghpages
+    echo "Done cleaning up!"
 }
 
+# Run this to verifiy that the docs build successfully.
+tox -e docs
```

Comment:
From where?

`tox -e docs` is the sphinx docs, and `buildDocs` generates the JSON files.

I know `docs` is in the tox envlist but we don't ever just call `tox` to run them all AFAIK.